src/objects/oo/zcl_abapgit_oo_serializer.clas.abap

```diff
@@ -251,9 +251,9 @@ CLASS zcl_abapgit_oo_serializer IMPLEMENTATION.
 
   METHOD serialize_abap_old.
 * for old ABAP AS versions
-    DATA: lo_source TYPE REF TO cl_oo_source.
+    DATA: lo_source TYPE REF TO object.
 
-    CREATE OBJECT lo_source
+    CREATE OBJECT lo_source TYPE ('CL_OO_SOURCE')
```

Comment:
this is the "serialize_abap_old", which is called when `CALL METHOD ('CL_OO_FACTORY')=>('CREATE_INSTANCE')` fails, so guess this change is okay

Reply:
Yes, I think it is only needed as fallback in 7.02.